CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [Next version]
 
+### Added
+- Public: `useLiveDocumentCapture` option added to the document capture step
+
```

Comment:
Are we not going to document this in the README?

Reply:
Not for now/this ticket. There is a follow up ticket where I'll discuss with See Wah how we want to document it in the README for beta testing.